myStyle.css

```diff
@@ -0,0 +1,38 @@
+  }
+div p {
+    color: navy;
+    margin-left: 20px;
+  }
+
+#ID {
+    background-color: red;
+  }
+
+.nice {
+    font-family: sans-serif;
+  }
+
+[target=_blank] {
+    background-color: yellow;
+  }
+
+p[target=_hi] {
```

Comment:
I think `target` is usually just used for `<a>` tags.
And if you really wanted to use `target` attribute, then you can just use `hi` as the value instead of `_hi`.

Lastly, `p[target=_hi]` can just be `[target=_hi]` like how line 17 doesn't need the `p` in front of `[target...]`.